casr/src/bin/casr-cli.rs

```diff
@@ -824,12 +824,16 @@ fn print_summary(dir: &Path, unique_crash_line: bool) {
             .map(|res| res.unwrap().path())
             .any(|e| e.extension().is_some() && e.extension().unwrap() == "casrep")
     {
-        clusters.push((dir.to_path_buf(), 0));
```

Comment:
canonicalize may fail

Reply:
Fixed, just try to canonicalize, push `dir` itself if failed